src/Microsoft.OData.Edm/Csdl/CsdlConstants.cs

```diff
@@ -180,6 +180,8 @@ public static class CsdlConstants
         internal const string Value_Ref = "Ref";
         internal const string Value_SridVariable = EdmConstants.Value_SridVariable;
         internal const string Value_ScaleVariable = EdmConstants.Value_ScaleVariable;
+        internal const string Value_SridVariable_Legacy = EdmConstants.Value_ScaleVariable_Legacy;

```

Comment:
Why not `Value_SridVariable_Legacy` just like there's separate constants for `Value_SridVariable` and `Value_ScaleVariable`

Reply:
yea that's a mistake. It should be `Value_SridVariable_Legacy`